integration-tests/fromMnemonic-send-tez.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { OperationContentsAndResultTransaction } from "@taquito/rpc";
+import { InMemorySigner } from "@taquito/signer";
+import { TezosToolkit } from "@taquito/taquito";
+import { CONFIGS } from "./config";
+
+CONFIGS().forEach(({ lib, rpc, setup }) => {
+  let Funder: TezosToolkit;
+   describe(`Test fromMnemonic instantiation with rpc: ${rpc}`, () => {
+    const mnemonic = 'author crumble medal dose ribbon permit ankle sport final hood shadow vessel horn hawk enter zebra prefer devote captain during fly found despair business'
```

Comment:
I think there will be an issue with this test if run simultaneously on different PRs; they will use the same signer and will create "counter" or "one-manager-per-block" errors. Can be solved by using a random mnemonic, bip39 has a method to generate mnemonics

Reply:
👍